include/netlink/route/nh.h

```diff
@@ -56,6 +56,19 @@ extern int rtnl_nh_get_family(struct rtnl_nh *);
 extern int rtnl_nh_set_group_type(struct rtnl_nh *, uint16_t group_type);
 extern int rtnl_nh_get_group_type(struct rtnl_nh *);
 
+/* Resilient nexthop group specific setters/getters */
+extern int rtnl_nh_set_res_group_bucket_size(struct rtnl_nh *,
+					     uint16_t buckets);
+extern int rtnl_nh_get_res_group_bucket_size(struct rtnl_nh *);
+
+extern int rtnl_nh_set_res_group_idle_timer(struct rtnl_nh *,
+					    uint32_t idle_timer);
+extern int rtnl_nh_get_res_group_idle_timer(struct rtnl_nh *);
+
+extern int rtnl_nh_set_res_group_unbalanced_timer(struct rtnl_nh *,
+						  uint32_t unbalanced_timer);
+extern int rtnl_nh_get_res_group_unbalanced_timer(struct rtnl_nh *);
```

Comment:
for `rtnl_nh_get_res_group_idle_timer()` and `rtnl_nh_get_res_group_unbalanced_timer()` it seems that the full range of values is `uint32_t`.

We cannot encode that in an `int` return value. The getters need to have a separate `uint32_t *out_value` parameter.